transformation/plugins/org.jboss.tools.fuse.transformation.editor/src/main/java/org/jboss/tools/fuse/transformation/editor/TransformationEditor.java

```diff
@@ -361,17 +370,74 @@ public void init(IEditorSite site,
             if (!latestVersion) prefs.setValue(VERSION_PREFERENCE, version);
 
             // Ensure Maven will compile transformations folder
-            final IProject project = manager.project();
+            IProject project = manager.project();
```

Comment:
The method TransformationEditor.init is 100 lines long. We shoudl split it for better readability.
For instance I see several distinct part (which can surely subdivided in soelm cases):
- create Transformation Folder
- ensure Pom contains MANIFEST.MF configuration
- compile project